code/game/machinery/computer/crew.dm

```diff
@@ -227,25 +228,32 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		if(pos.z != z && (!is_station_level(pos.z) || !is_station_level(z)) && !HAS_TRAIT(tracked_living_mob, TRAIT_MULTIZ_SUIT_SENSORS))
 			continue
 
+		var/sensor_mode
 		var/mob/living/carbon/human/tracked_human = tracked_living_mob
 
-		// Check their humanity.
+		// Set sensor level based on whether we're in the nanites list or the suit sensor list.
+		if(tracked_living_mob in GLOB.nanite_sensors_list)
+			sensor_mode = SENSOR_COORDS
+
 		if(!ishuman(tracked_human))
+			// Check their humanity.
 			stack_trace("Non-human mob is in suit_sensors_list: [tracked_living_mob] ([tracked_living_mob.type])")
 			continue
 
 		// Check they have a uniform
 		var/obj/item/clothing/under/uniform = tracked_human.w_uniform
-		if (!istype(uniform))
+		if (istype(uniform))
+			// Check if their uniform is in a compatible mode.
+			if((uniform.has_sensor == NO_SENSORS) || !uniform.sensor_mode)
+				stack_trace("Human without active suit sensors is in suit_sensors_list: [tracked_human] ([tracked_human.type]) ([uniform.type])")
+				continue
+
+			sensor_mode = uniform.sensor_mode
+		else
 			stack_trace("Human without a suit sensors compatible uniform is in suit_sensors_list: [tracked_human] ([tracked_human.type]) ([uniform?.type])")
 			continue
```

Comment:
I feel like there was meant to be some handling for having nanites here otherwise this might stack_trace over every naked nanite user